### PR TITLE
Update ManageWiki to new config format

### PR DESCRIPTION
**Pull request by paladox**

### CODE REVIEW

File: ManageWiki.php [hunk 1]
```diff
@@ -224,24 +213,25 @@
 			'linkPage' => 'https://mediawiki.org/wiki/Extension:AutoCreateCategoryPages',
 			'var' => 'wmgUseAutoCreateCategoryPages',
 			'conflicts' => false,
-			'requires' => false,
-			'restricted' => true,
```
Comment (JohnFLewis) [inline ManageWiki.php]: restricted

File: ManageWiki.php [hunk 2]
```diff
@@ -286,72 +274,72 @@
...
-			'restricted' => false,
+			'requires' => [],
 		],
 		'collection' => [
 			'name' => 'Collection (PDF)',
 			'linkPage' => 'https://mediawiki.org/wiki/Extension:Collection',
 			'var' => 'wmgUseCollection',
 			'conflicts' => false,
-			'requires' => 'electronpdfservice',
-			'restricted' => true,
```
Comment (JohnFLewis) [inline ManageWiki.php]: was this just restricted for the ext requirement?

File: ManageWiki.php (hunk 1 above)
Comment (paladox) [reply ManageWiki.php]: Thanks, done!

File: ManageWiki.php (hunk 2 above)
Comment (paladox) [reply ManageWiki.php]: Ah yes, i think so. This should now work (so no longer needs restricting).

**Pull request merged**